scripts/appleseed.package.py

```diff
@@ -190,6 +191,103 @@ def merge_tree(src, dst, symlinks=False, ignore=None):
+            subprocess.check_output(cmd, stderr=subprocess.STDOUT)
+
+    subprocess.check_call(['iconutil', '--convert', 'icns', icon_set, '-o', icon_path])
+    safe_delete_directory(icon_set)
+
+def create_mac_app(staging_dir, app_path, icon_path):
+
+    # cleanup old app
+    safe_delete_directory(app_path)
+
+    contents = os.path.join(app_path, "Contents")
+
+    shutil.move(staging_dir, contents)
+    # shutil.copytree(staging_dir, contents, symlinks=True)
+
+    resources = os.path.join(contents,"Resources")
+    safe_make_directory(resources)
+
+    macos =  os.path.join(contents,"MacOS")
+    safe_make_directory(macos)
+
+    # relative symlink for executable
+    os.symlink('../bin/appleseed-studio', os.path.join(macos, 'appleseed'))
+
+    icon = os.path.join(resources, "appleseed.icns")
+    create_mac_icon(icon_path, icon)
+
+    with open(os.path.join(contents, "Info.plist"), 'wb') as f:
+        f.write(b'<?xml version="1.0" encoding="UTF-8"?>\n')
+        f.write(b'<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">\n')
+        f.write(b'<plist version="1.0">\n')
+        f.write(b'<dict>\n')
+        f.write(b'  <key>NSPrincipalClass</key>\n')
+        f.write(b'  <string>NSApplication</string>\n')
+        f.write(b'  <key>CFBundlePackageType</key>\n')
+        f.write(b'  <string>APPL</string>\n')
+        f.write(b'  <key>CFBundleExecutable</key>\n')
+        f.write(b'  <string>appleseed</string>\n')
+        f.write(b'  <key>CFBundleIconFile</key>\n')
+        f.write(b'  <string>appleseed.icns</string>\n')
+        f.write(b'  <key>CFBundleIdentifier</key>\n')
+        f.write(b'  <string>net.appleseedhq.appleseed</string>\n')
+        f.write(b'  <key>CFBundleShortVersionString</key>\n')
+        f.write(b'  <string>2.1.0</string>\n')
+        f.write(b'</dict>\n')
+        f.write(b'</plist>\n')
+
+def create_mac_dmg(app_path, dmg_path, background_path, drive_icon_path):
+
+    drive_icon = 'appleseed-drive.icns'
+    create_mac_icon(drive_icon_path, drive_icon)
+
+    settings = {
+        "title": "appleseed",
+        "background":  background_path,
+        "icon": drive_icon,
+        "format": "UDZO",
+        "compression-level": 9,
+        "window": { "position": { "x": 100, "y": 100 },
+                    "size": { "width": 640, "height": 280 } },
+        "contents": [
+            { "x": 140, "y": 120, "type": "file", "path": app_path},
+            { "x": 500, "y": 120, "type": "link", "path": "/Applications" },
+        ]
+    }
+
+    settings_file = "settings.json"
+    with open(settings_file, 'wb') as f:
+        json.dump(settings, f)
+
+    output_dir = os.path.dirname(dmg_path)
+    if not os.path.exists(output_dir):
+        os.makedirs(output_dir)
+
+    # use dmgbuild to create disk image
+    # https://github.com/al45tair/dmgbuild
+    cmd = ['dmgbuild', '-s', settings_file , 'appleseed', dmg_path]
```

Comment:
Same question for `dmgbuild`.

Reply:
This should be the only new 3rd party dependency added, `pip install dmgbuild` will install it. There is a little blurb about it in the install message, is there anywhere else this documentation should be added?

Reply:
Nope, the short explanation when the script starts is all we have right now.